components/builder-index/EnablingConditionsSelector.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import classnames from 'classnames';
+
+class EnablingConditionsSelector extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      activeTab: "stakeholder-actor",
+    };
+  }
+
+  selectTab(key) {
+    this.setState({ activeTab: key });
+  }
+
+  render() {
+    const tabs = {
+      actor: "Actor",
```

Comment:
I see a deep dependency here...
Even if the category tree should not change, specific names could eventually do...